firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/ApkInstaller.java

```diff
@@ -83,9 +83,8 @@ void handleAppResume(Activity activity) {
     }
   }
 
-  Task<Void> installApk(String path) {
+  Task<Void> installApk(String path, Activity currentActivity) {
     synchronized (installTaskLock) {
-      Activity currentActivity = lifeCycleNotifier.getCurrentActivity();
       // This ensures that we save the state of the install if the app is backgrounded during
       // APK download
       if (currentActivity == null) {
```

Comment:
We no longer have to account for this case, since `currentActivity` will never be null.

Reply:
will check with manual test but will delete.